src/time/time_span.cr

```diff
@@ -104,6 +104,7 @@ struct TimeSpan
   def seconds
     (ticks % TicksPerMinute / TicksPerSecond).to_i32
   end
+  alias_method to_i, seconds
```

Comment:
I don't expect #to_i to return the seconds part of the range, but the total seconds minus milliseconds, something like `total_seconds.floor`